examples/java8/src/test/java/org/apache/beam/examples/snippets/SnippetsTest.java

```diff
@@ -42,7 +45,29 @@
 public class SnippetsTest implements Serializable {
 
   @Rule
-  public transient TestPipeline p = TestPipeline.create();
+  public final transient TestPipeline p = TestPipeline.create();
+
+  @Test
+  public void testModelBigQueryIO() {
+    // We cannot test BigQueryIO functionality in unit tests, therefore we limit ourselves
+    // to making sure the pipeline containing BigQuery sources and sinks can be built.
+    boolean runLocally = false;
```

Comment:
<!--new_thread; commit:ae660d95c3a81daa41104384866f652ae761a937; resolved:0-->
Why do we need branching below given that this is always false ?

Reply:
It just makes it easier for us to test locally. I'll add some instructions as comments to make it clear.